### ISSUE TITLE
Is there any MS-DOS compatibility?

### ISSUE THREAD

**Issue by desche42** [kelseyhightower/nocode#842]
I need to implement this repository in a MS-DOS machine.

**Comment by D10100111001** [comment]
Yes it is fully compatible

**Comment by green-coder** [comment]
Already fixed: Nocode recently supported Windows, which is backward compatible with MS-DOS.

**Comment by pabru** [comment]
@green-coder Not true. Windows is a superset of MS-DOS, not the other way around. We'll need to add proper MS-DOS support at some stage, otherwise we'll never reach a larger market share, There are literally dozens of computers running MS-DOS and compatible OSes out there!
